src/Retriever.jl

```diff
@@ -25,6 +51,28 @@ function check_for_updates()
     rt.check_for_updates()
 end
 
+"""
+```julia
+    get_dataset_names_upstream(keywords::String="", licenses::String="", repo::String="")
+```
+
+Get dataset names upstream scripts.
+"""
+function get_dataset_names_upstream(keywords::String="", licenses::String="", repo::String="")
+    rt.get_dataset_names_upstream(keywords, licenses, repo)
```

Comment:
`keywords` and `licenses` are lists in core. Should they be arrays or something similar here?